lib/src/mt_rtcp.c

```diff
@@ -263,7 +230,31 @@ int mt_rtcp_rx_send_nack_packet(struct mt_rtcp_rx* rx) {
   if (now < rx->nacks_send_time) return 0;
   rx->nacks_send_time = now + rx->nacks_send_interval;
 
-  if (MT_TAILQ_FIRST(&rx->nack_list) == NULL) return 0;
+  uint16_t num_fci = 0;
+  struct mt_rtcp_fci fcis[32];
+
+  /* check missing pkts with bitmap, fill fci fields */
+  uint16_t seq = rx->last_cont + 1;
+  uint16_t start = seq;
+  uint16_t miss = 0;
+  while (rtp_seq_num_cmp(seq, rx->last_seq - rx->seq_skip_window) < 0) {
```

Comment:
TBD, in some case we need to check miss without any skipping, like marker packet

Reply:
Next will consider handling st20 marker miss case